cypress/integration/tests/keycloak/rbac.architect.test.ts

```diff
@@ -18,10 +17,18 @@ describe("Keycloak operations", () => {
         preservecookies();
     });
 
-    it("Login as architect and validate permissions", () => {
+    it("Login as architect and validate create application button", () => {
```

Comment:
It doesn't login each time, session is saved. I thought you want to divide those tests so we will always know which of them failed, otherwise I have single method to call bunch of validation methods.